pythonforandroid/recipes/python3crystax/__init__.py

```diff
@@ -12,7 +12,7 @@
 
 
 class Python3Recipe(TargetPythonRecipe):
-    version = '3.5'
+    version = '3.6'
```

Comment:
I've bumped the default to 3.6 because Python 3.5 is getting older and 3.6 is the default Python3 version of various distributions including Ubuntu 18.04 LTS.
It's still possible to force specific version with `python3crystax==3.6` if needed.